packages/strapi-design-system/src/Avatar/Avatar.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+export const Avatar = styled.img`
```

Comment:
Should we add a PropTypes to make sure the alt is always defined?

Reply:
Yep, sure